packages/lynx-react/src/client.ts

```diff
@@ -14,7 +14,7 @@ import {
   addAutoIpAddressToSession,
   addAutoIpAddressToUser,
```

Comment:
m: `addAutoIpAddressToUser` was deprecated because it's the function that sets the wrong ip address inference field on `event.user`. We can probably just remove the import and the client hook further below in this file.